akka-stream-tests/src/test/scala/akka/stream/scaladsl/SourceSpec.scala

```diff
@@ -223,44 +223,53 @@ class SourceSpec extends AkkaSpec {
     val expected = List(9227465, 5702887, 3524578, 2178309, 1346269, 832040, 514229, 317811, 196418, 121393, 75025, 46368, 28657, 17711, 10946, 6765, 4181, 2584, 1597, 987, 610, 377, 233, 144, 89, 55, 34, 21, 13, 8, 5, 3, 2, 1, 1, 0)
 
     "generate a finite fibonacci sequence" in {
-      import GraphDSL.Implicits._
-
+      import system.dispatcher
       val source = Source.unfold((0, 1)) {
         case (a, _) if a > 10000000 ⇒ None
         case (a, b)                 ⇒ Some((b, a + b) → a)
       }
-      val result = Await.result(source.runFold(List.empty[Int]) { case (xs, x) ⇒ x :: xs }, 1.second)
-      result should ===(expected)
+      source.runFold(List.empty[Int]) { case (xs, x) ⇒ x :: xs } pipeTo testActor
+      expectMsg(expected)
```

Comment:
why? The error reporting by `should ===` is usually better, and `Await` will rethrow the onError value here, which is easier to parse than it being wrapped in a Status.Failure that was got instead of an expected message.